modules/react/button/lib/ExternalHyperlink.tsx

```diff
@@ -41,18 +42,29 @@ const StyledSystemIcon = styled(SystemIcon)<StyledType>({
 export const ExternalHyperlink = createComponent('a')({
   displayName: 'ExternalHyperlink',
   Component: (
-    {children, iconLabel = 'Opens link in new window', ...elemProps}: ExternalHyperlinkProps,
-    ref
+    {
+      children,
+      iconLabel = 'Opens link in new window',
```

Comment:
We should remove these default strings

Reply:
?

Reply:
The `iconLabel` prop should be added to all examples. Any defaulted prop like this means we have English embedded into our source code that can show up in the UI. We don't have a functional reason to default to English strings, right? We've been removing them elsewhere.

Reply:
I think that's the sentiment now right?